src/adapter/services/LegacyObjectAPIInterceptor.js

```diff
@@ -79,7 +79,7 @@ define([
         );
     }
     ObjectServiceProvider.prototype.update = async function (object) {
-        let model = utils.toOldFormat(object);
+        let model = await utils.toOldFormat(object);
```

Comment:
`utils.toOldFormat` is not an async call

Reply:
changed to awaiting a variable which then gets returned